src/routes/(split)/components/toasts/+page.md

```diff
@@ -63,6 +63,12 @@
         level: "info",
         duration: 3000
     });
+
+    const withHtml = () => toastsStore.show({
```

Comment:
How about adding an HTML tag in the multiline message above to demonstrate that it's rendered as text?

And are there no unit tests?

Reply:
I thought there were no tests here, my bad.

I preferred to add another button with html rendered as text. So that when reading the button it indicates simply what to expect and do only one thing.